bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -28,88 +26,87 @@ class TriviaQuiz(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
         self.questions = self.load_questions()
-        self.game_status = {}
-        self.game_owners = {}
+        self.game_status = {}  # A variable to store the game status: either running or not running.
```

Comment:
It strikes me as simpler to maintain a single object (probably a dataclass) to represent game state instead of 4 separate dictionaries. However, I acknowledge this has been in review for a while already so this change can be held off for now considering the current approach is functional.

Reply:
The old version of this quiz used a data class as there were many dictionaries but I had to ditch that code as I could not fix errors such as to reset the quiz game in a channel.
I tried a new solution given by Kosa which is working really good now and it does not require many dictionaries so thats the reason I am not using a data class.